static/js/common/upload-addon.js

```diff
@@ -27,7 +27,7 @@
     }
 
     $.fn.addonUploader = function( options ) {
-        var settings = {'filetypes': ['xpi', 'jar', 'xml'], 'getErrors': getErrors, 'cancel': $()};
+        var settings = {'filetypes': ['zip', 'xpi', 'jar', 'xml'], 'getErrors': getErrors, 'cancel': $()};
```

Comment:
According to https://github.com/mozilla/addons-server/blob/master/src/olympia/files/models.py#L36 `.json` is also allowed while we're at it. But, tbh I don't know if we want to change that :laughing: